.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,7 +30,9 @@ feature
 **What does this PR do / Why do we need it**:
 
 
-**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:
+<!--
```

Comment:
nit: let's move this comment to right below the above prompt, and we can fit it on one line